composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0|^8.1|^8.2|^8.3",
+        "php": "^8.0|^8.1|^8.2|^8.3",
```

Comment:
Did it throw error on 7.4? You should keep it.

Reply:
Sorry for not explaining
In the file [LaravelRequestDocsToOpenApi](https://github.com/rakutentech/laravel-request-docs/blob/dc003526eba0fe747197c309c2adbc6fd7308ed6/src/LaravelRequestDocsToOpenApi.php#L111), there is a mention of str_contains which was only added in PHP 8+ for this reason I removed the reference to PHP 7.4 from composer
Reference [str_contains](https://www.php.net/manual/en/function.str-contains.php)